alws/release_planner.py

```diff
@@ -554,6 +561,7 @@ async def get_release_plan(
                             if repo['arch'] == pkg['arch']:
                                 repo['arch'] = weak_arch
                         beholder_cache[second_key] = replaced_pkg
+                        logging.debug('beholder_cache: %s', str(beholder_cache))
```

Comment:
this line will flood our logs, move it to 572 line, after `if not` condition

Reply:
Good catch! Already fixed it. Sorry, I don't have a sense of how many times such loop can keep going :sweat_smile: